addons/point_of_sale/static/src/app/generic_components/alert_service.js

```diff
@@ -3,7 +3,7 @@ import { registry } from "@web/core/registry";
 
 class Alert extends Component {
     static template = xml`
-        <div t-attf-class="alert fixed-top p-1 rounded-0 alert-{{props.type}} fade show d-flex align-items-center justify-content-center" role="alert" style="height:48px">
+        <div t-attf-class="alert fixed-top p-1 rounded-0 alert-{{props.type}} fade show d-flex align-items-center justify-content-center" role="alert" t-attf-style="height: {{props.height}}px;">
```

Comment:
i think that you can just put here the current value for the height of the top bar. I don't see a usecase for having it customizable via props

Reply:
I don't know its height.

Reply:
what do you think about just giving it a hard size ? We can set it to `4rem` and then the alert service would be much simpler. The navbar is not really supposed to change it's height anyways

Reply:
I personally don't like the idea because once we start changing the height of the topheader, we'll have to adapt the height of this banner which we'll likely forget.

Reply:
i was thinking that we would have a css class for it that we would use in both places
i.e. ```css
.navbar-height {
   heigth: 4rem;
}
```

Reply:
yes, but it's height is never supposed to change